oioioi/teachers/utils.py

```diff
@@ -0,0 +1,51 @@
+from enum import Enum
+
+from oioioi.participants.models import Participant
+from oioioi.teachers.models import ContestTeacher, Teacher
+
+
+def get_user_teacher_obj(user):
+    try:
+        return user.teacher
+    except Teacher.DoesNotExist:
+        return None
+
+
+def is_user_already_in_contest(user, contest):
+    teacher = get_user_teacher_obj(user)
+
+    if user.participant_set.filter(contest=contest) or \
+            teacher and teacher.contestteacher_set.filter(contest=contest):
```

Comment:
this is valid, but I prefer to have brackets around (teacher and teacher.contest...)